### PR TITLE
fix: show cost_center in print format of report trial balance

### PR DESCRIPTION
**Pull request by Anurag810**
Added cost center if present in reports in print format.
Screenshot 2019-06-11 at 11 14 44 AM

### CODE REVIEW

**Review by adityahase — DISMISSED** [review]

**Comment by rohitwaghchaure** [comment]
@Anurag810 add screenshots of the print before and after of your change

**Comment by sahil28297** [comment]
@frappe-bot create instance

File: erpnext/accounts/report/financial_statements.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
 {% } %}
 <h2 class="text-center">{%= __(report.report_name) %}</h2>
 <h3 class="text-center">{%= filters.company %}</h3>
+<h3 class="text-center">{%= filters.cost_center %}</h3>
```
Comment (nabinhait) [inline erpnext/accounts/report/financial_statements.html]: Add this only if filtered by cost center.

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/accounts/report/financial_statements.html (hunk 1 above)
Comment (Anurag810) [reply erpnext/accounts/report/financial_statements.html]: ok

**Comment by sahil28297** [comment]
@frappe-bot create instance

**Comment by sahil28297** [comment]
@frappe-bot create instance

**Comment by sahil28297** [comment]
@frappe-bot create instance

**Comment by sahil28297** [comment]
@frappe-bot create instance

**Comment by sahil28297** [comment]
@frappe-bot stop instance

**Pull request merged**